qiita_db/test/test_util.py

```diff
@@ -1291,6 +1291,12 @@ def test_purge_filepaths_test(self):
         fps_viewed = self._get_current_filepaths()
         self.assertCountEqual(fps_expected, fps_viewed)
 
+    def test_quick_mounts_purge(self):
+        # one of the tests creates a conflicting artifact_type so this test
```

Comment:
Just to confirm, you're relying on state modified by another test for this test to work properly. That does seem to go against the idea of each test being independent and state being reset between tests. I'm okay with it, but how do we ensure that the test that creates the conflicting artifact_type is always called before this test?

Reply:
Kind of, the issue is that other tests start and restart the database multiple times and some of them leave behind some artifacts so the test folder is populated with multiple versions of the artifacts - which we had no idea about. In specific, some leftover artifacts have the same id (2) in different mounts (in specific Demultiplexed and per-sample-FASTQ). I really don't have a strong opinion about fixing it but please open an issue to address that if you feel like we should fix.

Reply:
I think as long as it passes testing that's good enough for me.